### ISSUE TITLE
OnInfiniteLoad not called when scrolled in parent

### ISSUE THREAD

**Issue by raksheetbhat** [jsdevkr/react-chatview#35]
Hey, I've run into this issue where onInfinteLoad is not called from the parent component. Basically, I have react-chatview in a child component and the onScroll in the chatview child is not getting triggered when scrolled in the parent. An earlier workaround I had tried was to set the position of the child as fixed or absolute, which ensured that the scroll is triggered in the child. But, I don't want to use position as fixed.

What should I do to fix this?

One solution I feel can work is to manually trigger the react-chatview's onScroll component when scroll happens in the parent. But I feel this might complicate stuff. Please let me know if anything else can be done.

Thanks.

**Comment by gimdongwoo** [comment]
It is not easy to understand for me what the problem is.
I can make improvements by submitting the sample code of you.

**Comment by gimdongwoo** [comment]
I think you can check `shouldTriggerLoad` prop.
`onInfiniteLoad` trigging need to `shouldTriggerLoad()` results is true.
You can add `shouldTriggerLoad={() => { return true; }}`

**Comment by raksheetbhat** [comment]
Okay thanks

**Comment by raksheetbhat** [comment]
Thanks for your reply, and for your suggestions. I actually did find a way to circumvent the problem yesterday. So my exact problem was that, from your JS fiddle example, you had put the react-chatview container in a position: fixed, which fixed the chatview on top of the parent component But I wanted to get chatview working without fixing its position.

The workaround was to manually trigger the loadMore function when I scroll to the bottom of the parent class. 

I used el.getBoundingClientRect().bottom <= window.innerHeight to detect scroll to the bottom.
